specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json

```diff
@@ -495,19 +493,683 @@
         },
         "confidenceScore": {
           "format": "float",
-          "x-ms-client-name": "confidence",
           "description": "The entity confidence score.",
           "type": "number"
         },
-        "listKeys": {
+        "resolutions": {
           "type": "array",
-          "description": "List of keys.",
+          "items": {
+            "$ref": "#/definitions/BaseResolution"
+          }
+        },
+        "extraInformation": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BaseExtraInformation"
+          }
+        }
+      }
+    },
+    "BaseExtraInformation": {
+      "type":"object",
+      "discriminator": "extraInformationKind",
+      "properties": {
+        "extraInformationKind":{
+          "type": "string",
+          "enum": [
+            "EntitySubtype",
+            "ListKeys"
```

Comment:
If you changed the property to "resolutions", shouldn't this change as well?

Reply:
It's not just a rename. `ListKeys` in this new design, inherits from the abstract "BaseExtraInformation". The Entity response has an array of polymorphic objects for ExtraInformation that can contain objects of the type `ListKeys` and `EntitySubtype`.